flake.nix

```diff
@@ -29,10 +31,9 @@
         hm = import ./modules/lib { lib = nixpkgs.lib; };
         homeManagerConfiguration = { configuration, system, homeDirectory
           , username, extraModules ? [ ], extraSpecialArgs ? { }
-          , pkgs ? builtins.getAttr system nixpkgs.outputs.legacyPackages
-          , check ? true, stateVersion ? "20.09" }@args:
+          , pkgs ? nixpkgsFor.${system}, check ? true, stateVersion ? "20.09"
```

Comment:
This is a nit, but why do we need to use a custom function when `nixpkgs.legacyPackages.${system}` would suffice?

Reply:
Figured it was a more succinct expression that was being used elsewhere anyways. I can revert this particular change, though.